xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3627,7 +3627,7 @@ platform.linkchecker.livetable.date=Last Checked
 
 ### Dashboard Application Resources
 platform.dashboard.user.preferences=Dashboard preferences
-platform.dashboard.user.displayOnMainPage=Display my dashboard on the wiki home when I'm logged in (instead of the default dashboard)
+platform.dashboard.user.displayOnMainPage=Display my dashboard when I'm logged in (instead of the default dashboard)
```

Comment:
hmm I think it can be improved, it looks a bit strange.
What about: "Display my dashboard when viewing the Dashboard application" which is what it does AFAIK?